### PR TITLE
edits to README

### PR DESCRIPTION
**Pull request by jbondpdx**
Just some edits to new or changed material.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -173,8 +170,8 @@ Converts a boolean to a number. Converts values:
 #### `bool2str`
 
 Converts a boolean to a string using optionally supplied arguments. The optional
-second and third arguments represent what true and false will be converted to
-respectively. If only one argument is given, it will be converted from a boolean
+second and third arguments represent what true and false are converted to
```
Comment (bmjen) [inline README.markdown]: unnecessary carriage return

File: README.markdown [hunk 2]
```diff
@@ -120,20 +116,21 @@ Match Example With `ensure => absent`:
...
     }
 
-In this code example match will look for a line beginning with export
+In this code example, `match` looks for a line beginning with export
 followed by HTTP_PROXY and delete it.  If multiple lines match, an
 error will be raised unless the `multiple => true` parameter is set.
 
-**Autorequires:** If Puppet is managing the file that will contain the line
-being managed, the file_line resource will autorequire that file.
+**Autorequires:** If Puppet is managing the file that contains the line
```
Comment (bmjen) [inline README.markdown]: unnecessary carriage return

File: README.markdown [hunk 3]
```diff
@@ -107,8 +103,8 @@ Match Example:
       match  => '^export\ HTTP_PROXY\=',
     }
 
-In this code example match will look for a line beginning with export
-followed by HTTP_PROXY and replace it with the value in line.
+In this code example, `match` looks for a line beginning with export
```
Comment (bmjen) [inline README.markdown]: unnecessary carriage return

File: README.markdown [hunk 4]
```diff
@@ -95,8 +91,8 @@ Example:
       line => '%sudonopw ALL=(ALL) NOPASSWD: ALL',
     }
 
-In this example, Puppet will ensure both of the specified lines are
-contained in the file /etc/sudoers.
+In this example, Puppet ensures that both of the specified lines are
```
Comment (bmjen) [inline README.markdown]: unnecessary carriage return

File: README.markdown [hunk 5]
```diff
@@ -120,20 +114,20 @@ Match Example With `ensure => absent`:
       match_for_absence => true,
     }
 
-In this code example match will look for a line beginning with export
+In this code example, `match` looks for a line beginning with export
```
Comment (bmjen) [inline README.markdown]: unnecessary carriage return

File: README.markdown [hunk 6]
```diff
@@ -1160,7 +1142,7 @@ test, and the second argument should be a stringified regular expression (withou
   validate_re($::puppetversion, '^2.7', 'The $puppetversion fact value does not match 2.7')
   ~~~
 
-  Note: Compilation will also abort, if the first argument is not a String. Always use
+  Note: Compilation terminates if the first argument is not a string. Always use
```
Comment (bmjen) [inline README.markdown]: unnecessary carriage return

**Comment by igalic** [comment]
i don't understand.

how will people with text editors from the seventies now read / write this document?

**Comment by jbondpdx** [comment]
Oh, @igalic ... always thinking of others. /hug

**Pull request merged**